src/org/labkey/test/tests/TriggerScriptTest.java

```diff
@@ -804,4 +838,27 @@ private void setupDataClass() throws CommandException, IOException
                         new FieldDefinition(COUNTRY_FIELD, FieldDefinition.ColumnType.String)));
         dataClass.create(createDefaultConnection(), getProjectName());
     }
+
+    /**
+     * Setup the sample type
+     */
+    private void setupSampleType() throws CommandException, IOException
+    {
+        //Setup Sample Type
+        goToProjectHome();
+        DataRegionTable drt = DataRegionTable.findDataRegionWithinWebpart(this, "Sample Types");
+        int rowId = drt.getRowIndex("Name", SAMPLE_TYPE_NAME);
+        if (rowId >= 0)
+        {
+            drt.checkCheckbox(rowId);
+            drt.clickHeaderButtonAndWait("Delete");
+            clickButton("Confirm Delete");
+        }
+
+        SampleTypeDefinition sampleType = new SampleTypeDefinition(SAMPLE_TYPE_NAME)
+                .setFields(List.of(
+                        new FieldDefinition(COMMENTS_FIELD, FieldDefinition.ColumnType.String),
+                        new FieldDefinition(COUNTRY_FIELD, FieldDefinition.ColumnType.String)));
+        sampleType.create(createDefaultConnection(), getProjectName());
```

Comment:
`SampleTypeAPIHelper.createEmptySampleType` does this fully by API.
```suggestion
        SampleTypeDefinition sampleType = new SampleTypeDefinition(SAMPLE_TYPE_NAME)
                .setFields(List.of(
                        new FieldDefinition(COMMENTS_FIELD, FieldDefinition.ColumnType.String),
                        new FieldDefinition(COUNTRY_FIELD, FieldDefinition.ColumnType.String)));
        SampleTypeAPIHelper.createEmptySampleType(getProjectName(), sampleType);
```

Reply:
done